coretex/cli/resources/update_script_template.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+# Define Docker-related variables
+DOCKER_PATH={dockerPath}
+REPOSITORY={repository}
+TAG={tag}
+IMAGE={repository}:{tag}
+SERVER_URL={serverUrl}
+STORAGE_PATH={storagePath}
+NODE_ACCESS_TOKEN={nodeAccessToken}
+CONTAINER_NAME={containerName}
+NETWORK_NAME={networkName}
+RESTART_POLICY={restartPolicy}
+PORTS={ports}
+CAP_ADD={capAdd}
+RAM_MEMORY={ramMemory}G
+SWAP_MEMORY={swapMemory}G
+SHARED_MEMORY={sharedMemory}G
+
+# Main execution
+NODE_UPDATE_COMMAND="update_node"
+
+NODE_STATUS_ENDPOINT="http://localhost:21000/status"
+
+# Dump logs to CLI directory
+OUTPUT_DIR="$HOME/.config/coretex"
+mkdir -p "$OUTPUT_DIR"
+
+# Generate the output filename based on the current Unix timestamp
+OUTPUT_FILE="$OUTPUT_DIR/ctx_autoupdate.log"
+
+# Redirect all output to the file
+exec >>"$OUTPUT_FILE" 2>&1
+
+function should_update {{
+    local api_response=$(curl -s "$NODE_STATUS_ENDPOINT")
+    local status=$(echo "$api_response" | sed -n 's/.*"status":\([^,}}]*\).*/\1/p')
```

Comment:
This is called in 2 places, why not extract it into a separate function which returns Node status?

Reply:
This was not replaced with `fetch_node_status` call